operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -12,18 +12,20 @@ With {rh-rhacs-first}, you can scan RHCOS nodes for vulnerabilities and detect p
 
 {product-title-short} scans RHCOS RPMs installed on the node host, as part of the RHCOS installation, for any known vulnerabilities.
 
-First, {product-title-short} analyzes and detects RHCOS components, and then it matches vulnerabilities for identified components using Red Hat RHEL and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
+First, {product-title-short} analyzes and detects RHCOS components, and then it matches vulnerabilities for identified components by using Red Hat RHEL and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
 
 [NOTE]
 ====
-* Red Hat supports scanning for RHCOS nodes only if you are using it with {ocp}.
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* If you have installed {product-title-short} by using `roxctl` CLI, you must manually enable the RHCOS node scanning features. When you use Helm or Operator installation methods on {ocp}, this feature is enabled by default.
+* Red Hat supports scanning for RHCOS nodes only if you are using it with {ocp} 4.10 and later.
```

Comment:
Consider clarifying what "it" is in this sentence. For example, is this correct? "Red Hat supports scanning for RHCOS nodes only if you are using RHCOS with {ocp} 4.10 and later."